acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/clique/CliqueMiningAcceptanceTest.java

```diff
@@ -103,21 +110,40 @@ public void shouldMineTransactionsOnMultipleNodes() throws IOException {
 
   @Test
   public void shouldStallMiningWhenInsufficientValidators() throws IOException {
-    final BesuNode minerNode1 = besu.createCliqueNode("miner1");
-    final BesuNode minerNode2 = besu.createCliqueNode("miner2");
-    final BesuNode minerNode3 = besu.createCliqueNode("miner3");
+    final BesuNode minerNode1 = besu.createCliqueNode("miner1", CLIQUE_OPTIONS);
+    final BesuNode minerNode2 = besu.createCliqueNode("miner2", CLIQUE_OPTIONS);
+    final BesuNode minerNode3 = besu.createCliqueNode("miner3", CLIQUE_OPTIONS);
     startClusterAndVerifyProducingBlocks(minerNode1, minerNode2, minerNode3);
 
-    cluster.stopNode(minerNode2);
-    cluster.stopNode(minerNode3);
+    LOG.info("Checking active validators before removing stopped nodes...");
+    LOG.info("Active Validators BEFORE removal: {}", minerNode1.execute(cliqueTransactions.createGetSigners("latest")));
+
+// Propose to remove validators
```

Comment:
need the initial validators

    final String[] initialValidators = {"miner1"};
    final BesuNode minerNode1 =
            besu.createCliqueNodeWithValidators("miner1", CLIQUE_OPTIONS, initialValidators);
    final BesuNode minerNode2 =
            besu.createCliqueNodeWithValidators("miner2", CLIQUE_OPTIONS, initialValidators);
    final BesuNode minerNode3 =
            besu.createCliqueNodeWithValidators("miner3", CLIQUE_OPTIONS, initialValidators);
    cluster.start(minerNode1, minerNode2, minerNode3);

and then add miner2 and 3 before removing them